common/src/chain/pos.rs

```diff
@@ -124,20 +127,23 @@ pub fn create_unittest_pos_config() -> PoSChainConfig {
         reward_maturity_distance: 2000.into(),
         decommission_maturity_distance: 2000.into(),
         spend_share_maturity_distance: 2000.into(),
-        block_count_to_average_for_blocktime: 5,
-        difficulty_change_limit: PerThousand::new(100).expect("must be valid"),
+        block_count_to_average_for_blocktime: 100,
```

Comment:
If this is the same number everywhere, please use a `const` somewhere for it. Same for others.